tests/openvino/native/models.py

```diff
@@ -225,3 +225,34 @@ def _create_ov_model(self):
         r1 = opset.result(add, name="Result_Add")
         model = ov.Model([r1], [input_1])
         return model
+
+
+@SYNTHETIC_MODELS.register()
```

Comment:
If you register `DynamicModel` as `@SYNTHETIC_MODELS`, you must also add `tests/openvino/native/data/reference_scales`.

Reply:
Good catch, thanks! Fixed.